lib/linked_in/access_token.rb

```diff
@@ -11,9 +11,11 @@ class AccessToken
     # @param [String] token the access token
     # @param [FixNum] expires_in number of seconds the token lasts for
     # @param [Time] expires_at when the token will expire.
-    def initialize(token=nil, expires_in=nil, expires_at=nil)
+    def initialize(token = nil, expires_in = nil, expires_at = nil, refresh_token = nil, refresh_token_expires_in = nil)
       self.token = token
       self.expires_in = expires_in
+      self.refresh_token = refresh_token
+      self.refresh_token_expires_in = refresh_token_expires_in
```

Comment:
Are these used internally?

Reply:
@radiantshaw currently we are not using it but added it if we initialize the `AccessToken` object so it should contain refresh token as well from now on.